cinn/optim/ir_copy.cc

```diff
@@ -23,6 +23,7 @@
 #include "cinn/common/ir_util.h"
 #include "cinn/ir/ir_mutator.h"
 #include "cinn/ir/ir_printer.h"
+#include "cinn/ir/ir_schedule.h"
```

Comment:
What's the purpose for changes in this file?

Reply:
same as explanation  in `ir_copy.h`, this is part of the solution